llpc/context/llpcCompiler.cpp

```diff
@@ -2331,6 +2331,12 @@ std::unique_ptr<Module> Compiler::createGpurtShaderLibrary(Context *context) {
   return module;
 }
 
+template <typename T> inline T *voidPtrIncAligned(const void *p, size_t numBytes) {
+  int8_t *ptr = reinterpret_cast<int8_t *>(const_cast<void *>(p));
+  ptr += (reinterpret_cast<intptr_t>(ptr) + sizeof(T) - 1) % sizeof(T);
```

Comment:
With the alternative solution, this becomes unnecessary. Still:

1. Should still add `numBytes`
2. Should use `alignof`, not `sizeof`
3. Need to add the bytes missing to the next multiple, e.g. `(-(uintptr_t)ptr) % alignof(T)`
4. This should be in `vkgcUtil.h` next to `voidPtrInc`